### PR TITLE
Fixed empty state dashboard issues

### PR DESCRIPTION
**Pull request by famole**
Empty state margin issues and missing text, still working in components reorder usign Flex.
Will be sent in a different PR

### CODE REVIEW

File: src/views/Dashboard/emptyState.tsx [hunk 1]
```diff
@@ -13,6 +13,10 @@ export const DashboardEmptyState: React.FC<{
   onClick: MouseEventHandler;
   props?: CenterProps;
 }> = ({ type, onClick, ...props }) => {
+  const renderType = React.useMemo(() => {
+    return type === EMPTY_TYPE.Deposit ? "deposited" : "borrowed";
+  }, [type]);
```
Comment (zedkai) [inline src/views/Dashboard/emptyState.tsx:18]: This is short enough - and, unlike objects and arrays, both types are primitives and are reference-equal; as a result, `React.useMemo` is unnecessary here, but it doesn't really hurt either. Feel free to leave it, but do try to note why it's unnecessary, and thus why a simple `const` would work.

File: src/views/Dashboard/emptyState.tsx [hunk 2]
```diff
@@ -24,20 +28,28 @@ export const DashboardEmptyState: React.FC<{
...
+        color="white"
+        mt="0.5rem"
+        fontWeight="400"
+        fontSize="smaller"
+        textAlign="center"
+        mx={{ base: "1rem", md: "3rem" }}
+      >
+        There will be a list of all the assets you have {renderType}.
+        For now, it’s empty since you have not {renderType}.
+      </Text>
```
Comment (zedkai) [inline src/views/Dashboard/emptyState.tsx]: Don't undo my changes ^^;
(See beccd104aced2a14b1050686e490954704573262)
```suggestion
```

**Review by zedkai — CHANGES_REQUESTED** [review]

File: src/views/Dashboard/emptyState.tsx (hunk 1 above)
Comment (famole) [reply src/views/Dashboard/emptyState.tsx:18]: thanks for the explanation, I wasn't sure about using it here.

**Review by zedkai — APPROVED** [review]
Applied the change myself.

**Pull request merged**